ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/builders.kt

```diff
@@ -42,8 +42,9 @@ public suspend fun HttpClient.webSocketSession(
                 val sessionCompleted = CompletableDeferred<Unit>()
                 sessionDeferred.complete(session)
                 session.outgoing.invokeOnClose {
-                    if (it != null) sessionCompleted.completeExceptionally(it)
-                    else sessionCompleted.complete(Unit)
+                    if (it != null) {
```

Comment:
i really don't like this formatting. maybe replace such changes with `when`?

Reply:
yep, we can

Reply:
Or just wrap the else expression with curly brackets